fdbserver/ApplyMetadataMutation.h

```diff
@@ -42,411 +42,10 @@ struct applyMutationsData {
 	Reference<KeyRangeMap<Version>> keyVersion;
 };
 
-static Reference<StorageInfo> getStorageInfo(UID id, std::map<UID, Reference<StorageInfo>>* storageCache, IKeyValueStore* txnStateStore) {
-	Reference<StorageInfo> storageInfo;
-	auto cacheItr = storageCache->find(id);
-	if(cacheItr == storageCache->end()) {
-		storageInfo = Reference<StorageInfo>( new StorageInfo() );
-		storageInfo->tag = decodeServerTagValue( txnStateStore->readValue( serverTagKeyFor(id) ).get().get() );
-		storageInfo->interf = decodeServerListValue( txnStateStore->readValue( serverListKeyFor(id) ).get().get() );
-		(*storageCache)[id] = storageInfo;
-	} else {
-		storageInfo = cacheItr->second;
-	}
-	return storageInfo;
-}
-
-// It is incredibly important that any modifications to txnStateStore are done in such a way that
-// the same operations will be done on all proxies at the same time. Otherwise, the data stored in
-// txnStateStore will become corrupted.
-static void applyMetadataMutations(UID const& dbgid, Arena &arena, VectorRef<MutationRef> const& mutations, IKeyValueStore* txnStateStore, LogPushData* toCommit, bool *confChange, Reference<ILogSystem> logSystem = Reference<ILogSystem>(), Version popVersion = 0,
+Reference<StorageInfo> getStorageInfo(UID id, std::map<UID, Reference<StorageInfo>>* storageCache, IKeyValueStore* txnStateStore);
+ 
+void applyMetadataMutations(UID const& dbgid, Arena &arena, VectorRef<MutationRef> const& mutations, IKeyValueStore* txnStateStore, LogPushData* toCommit, bool *confChange, Reference<ILogSystem> logSystem = Reference<ILogSystem>(), Version popVersion = 0,
 	KeyRangeMap<std::set<Key> >* vecBackupKeys = NULL, KeyRangeMap<ServerCacheInfo>* keyInfo = NULL, std::map<Key, applyMutationsData>* uid_applyMutationsData = NULL, RequestStream<CommitTransactionRequest> commit = RequestStream<CommitTransactionRequest>(),
```

Comment:
I kind of preferring using "nullptr" instead of NULL. Up to you if you want to change them.

Reply:
I'd request to do `NULL -> nullptr` change across all codebase in one commit to avoid mixed usage and thus future confusion of which is preferred.

Reply:
Maybe we can schedule a sprint/week to do spring-cleaning only changes, like clang-format or something similar.